src/libraries/System.Security.Cryptography.Pkcs/tests/EnvelopedCms/DecryptTestsUsingCertWithPrivateKey.cs

```diff
@@ -67,6 +67,31 @@ public static void DecryptMultipleRecipients()
             Assert.Equal<byte>(content, contentInfo.Content);
         }
 
+        [Fact]
+        public static void DecryptSuccesfullyWithWrongKeyProducesInvalidSymmetricKey()
+        {
+            using (X509Certificate2 wrongRecipient = Certificates.RSAKeyTransfer5_ExplicitSkiOfRSAKeyTransfer4.TryGetCertificateWithPrivateKey())
+            {
+                string encryptedContent =
```

Comment:
Want to leave a comment as to how you came up with this payload so someone can replace it/enhance it in the future (trial and error, manually encrypting a facetious payload and patching it in, etc)?

Reply:
Yeah, like https://github.com/dotnet/runtime/pull/50262#discussion_r601872422. But in the code :smile: